src/intervals/arithmetic/power.jl

```diff
@@ -120,11 +120,11 @@ end
 
 function ^(a::Interval{Rational{T}}, x::AbstractFloat) where {T<:Integer}
     a = Interval{Float64}(inf(a).num/inf(a).den, sup(a).num/sup(a).den)
-    return F(a^x)
+    return a^x
```

Comment:
Suggestion on line 122: imposing `a` to be `Interval{Float64}` seems limiting (to me) here. For the construction of `a`, I think we should rather use the type obtained by promoting `T` and `IntervalArithmetic.default_bound()`, or the actual type of `x`. Another option, I learned in your code below, is to use `float(Rational{T})`.

Reply:
That is a good point. I fixed `^`.